src/map/unit.c

```diff
@@ -466,126 +501,166 @@ static int unit_walktoxy_timer(int tid, int64 tick, int id, intptr_t data)
 				clif->fixpos(bl);
 				unit->attack(bl, tbl->id, ud->state.attack_continue);
 			}
-		} else { //Update chase-path
-			unit->walktobl(bl, tbl, ud->chaserange, ud->state.walk_easy|(ud->state.attack_continue? 1 : 0));
+		} else { // Update chase-path
+			unit->walktobl(bl, tbl, ud->chaserange, ud->state.walk_easy|(ud->state.attack_continue ? 1 : 0));
```

Comment:
```suggestion
			unit->walktobl(bl, tbl, ud->chaserange, ud->state.walk_easy | ud->state.attack_continue);
```
No reason for that inline-if, since `ud->state.attack_continue` can only be `1` or `0`.

Reply:
Done.